app/views/travel_advice/index.html.erb

```diff
@@ -30,13 +30,18 @@
                 <strong>
                   Under current UK COVID-19 restrictions, you must stay at home. You must not travel, including abroad, unless you have a legally permitted reason to do so. It is illegal to travel abroad for holidays and other leisure purposes.
                 </strong>
+              </p>
+              <p class="govuk-body">
                 Check the <a href="/guidance/national-lockdown-stay-at-home">rules that apply to you in England</a>, <a href="https://www.gov.scot/coronavirus-covid-19/">Scotland</a>, <a href="https://gov.wales/covid-19-alert-levels">Wales</a> and <a href="https://www.nidirect.gov.uk/articles/coronavirus-covid-19-regulations-guidance-what-restrictions-mean-you">Northern Ireland</a>.
               </p>
               <p class="govuk-body">
-                If you intend to <a href="/guidance/coronavirus-covid-19-testing-for-people-travelling-to-england">travel to England</a>, <a href="https://www.gov.scot/publications/coronavirus-covid-19-public-health-checks-at-borders/pages/testing-for-people-travelling-to-scotland/">Scotland</a>, <a href="https://gov.wales/coronavirus-covid-19-testing-people-travelling-wales">Wales</a>, or <a href="https://www.nidirect.gov.uk/articles/coronavirus-covid-19-international-travel-advice">Northern Ireland</a> from abroad, including UK nationals returning home, you must provide evidence of a negative COVID-19 test result taken up to 3 days before departure. If you do not comply (and you do not have a valid exemption) your airline or carrier may refuse you boarding and/or you may be fined on arrival.
+                If you intend to <a href="/guidance/coronavirus-covid-19-testing-for-people-travelling-to-england">travel to the UK from abroad</a>, including UK nationals returning home, you must provide evidence of a negative COVID-19 test result taken up to 3 days before departure. If you do not comply (and you do not have a valid exemption) your airline or carrier may refuse you boarding and/or you may be fined on arrival.
```

Comment:
Does this need a gov-link class?  Although the others don't have one 🤔

Reply:
No idea! I followed the existing pattern.